node/runtime/pangoro/src/lib.rs

```diff
@@ -134,9 +134,9 @@ frame_support::construct_runtime!(
 		Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>} = 12,
 		Grandpa: pallet_grandpa::{Pallet, Call, Storage, Config, Event} = 13,
 		Beefy: pallet_beefy::{Pallet, Storage, Config<T>} = 27,
-		BeefyGadget: darwinia_beefy_gadget::{Pallet, Call, Storage, Config} = 30,
-		Mmr: pallet_mmr::{Pallet, Storage} = 28,
-		MmrLeaf: pallet_beefy_mmr::{Pallet, Storage} = 29,
+		// BeefyGadget: darwinia_beefy_gadget::{Pallet, Call, Storage, Config} = 30,
+		// Mmr: pallet_mmr::{Pallet, Storage} = 28,
+		// MmrLeaf: pallet_beefy_mmr::{Pallet, Storage} = 29,
```

Comment:
I am not so familiar with these pallets' logic, I have some concerns about disabling these pallets in this way.  Are there any extra operations for these pallets' storage?

Reply:
#1277